graphdatascience/session/dedicated_sessions.py

```diff
@@ -109,10 +106,8 @@ def _find_existing_session(self, session_name: str) -> Optional[SessionDetails]:
 
         # this will only occur for admins as we cannot resolve Aura-API client_id -> console_user_id we fail for now
         if len(matched_sessions) > 1:
-            users = [s.user_id for s in matched_sessions]
             raise RuntimeError(
-                "Admin users need to chose a unique session name for now."
-                f" Multiple sessions with the name `{session_name}` across multiple users exist. Sessions found for users: {users}"
+                f"The use has access to multiple session with the name `{session_name}`. Please specify the id of the session that should be deleted."
```

Comment:
should we include the ids in the error message? also: typo
```suggestion
                f"The user has access to multiple session with the name `{session_name}`: `{[s.id for s in matched_sessions]}`. Please specify the id of the session that should be deleted."
```

Reply:
I don't think there is much use to listing the session ids here. The user will have to list sessions to find the correct one.